dependencies/precise/logify/control

```diff
@@ -0,0 +1,15 @@
+Source: ruby-logify
+Section: ruby
+Priority: optional
+Maintainer: Michael Moll <mmoll@mmoll.at>
+Build-Depends: debhelper, gem2deb
+Standards-Version: 3.9.5
+Homepage: https://github.com/sethvargo/logify
+XS-Ruby-Versions: all
+
+Package: ruby-logify
+Architecture: all
+XB-Ruby-Versions: ${ruby:Versions}
+Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, 
+Description: lightweight logging library for Ruby
+ Logify is an incredibly light-weight Ruby logger with a developer-friendly API and no dependencies. It is intentionally very opinionated and is optimized for speed. This combination makes it perfect for command line applications.
```

Comment:
This is intended, see
https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Description